subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -549,6 +549,12 @@ static void cdc_acm_tx_fifo_handler(struct k_work *work)
 	len = ring_buf_get(data->tx_fifo.rb, buf->data, buf->size);
 	net_buf_add(buf, len);
 
+	if (IS_ENABLED(CONFIG_USBD_CDC_ACM_ENSURE_SHORT_PKT_FOR_LAST_TX_PKT)) {
+		if (len != 0 && len % cdc_acm_get_bulk_mps(c_data) == 0) {
```

Comment:
```suggestion
		if ((len != 0) && ((len % cdc_acm_get_bulk_mps(c_data)) == 0)) {
```

Reply:
Thanks @ycsin for the suggestions. Have pushed the change accordingly.

Reply:
You should not follow this suggestion, please remove unnecessary parentheses.

Reply:
> You should not follow this suggestion, please remove unnecessary parentheses.

Isn't that MISRA C rule 12.1 & 13.5?

Reply:
Hi @ycsin, @jfischer-no, looked around the coding style for `usbd` and it seems that the style is more towards minimizing parentheses. 
Hence, for now I think it's better to remove the parentheses for consistency within `usbd`. 
Let's move on with this, we would need the ZLP change to be merge in ASAP so that we could resolve this issue with windows USBSER driver.

Reply:
🤷‍♂️ we can leave it for the misra people